### ISSUE TITLE
Socket Connections

### ISSUE THREAD

**Issue by shariqs** [jamesisaac/react-native-background-task#66]
Can I use this library to keep socket connections open?

**Comment by fauzynurn** [comment]
yes, i wanna know the answers. i implemented socket.io for react native and it kept disconnected whenever i locked the phone.

**Comment by kiranNegiloni** [comment]
> yes, i wanna know the answers. i implemented socket.io for react native and it kept disconnected whenever i locked the phone.

Hi @fauzynurn, I am having a similar issue where i need to keep the socket connection active. Please let me know your solution if you had already achieved it.
